### ISSUE TITLE
wrong alpha calculation?

### ISSUE THREAD

**Issue by pmcdonal** [adematti/cosmopipe#4]
Shouldn't alpha weighting include FKP weight? I.e., we're trying to make weighted randoms represent mean of weighted field as well as possible and, to the extent that it matters, FKP weight is part of that... 
https://github.com/adematti/cosmopipe/blob/a8d2e3c60ff9a892046ddab229ee13d2097e8312/cosmopipe/estimators/utils.py#L61

**Comment by adematti** [comment]
Yes, I agree at 100%. The reason I used this placeholder definition was to match nbodykit's: https://github.com/bccp/nbodykit/blob/4aec168f176939be43f5f751c90363b39ec6cf3a/nbodykit/algorithms/convpower/catalogmesh.py#L170
where data.W, randoms.W are the sums of completeness weights https://github.com/bccp/nbodykit/blob/4aec168f176939be43f5f751c90363b39ec6cf3a/nbodykit/algorithms/convpower/catalogmesh.py#L225
I did correct that for eBOSS; for DESI I think I'll update/build on nbodykit's ConvolvedFFTPower to e.g. tackle cross-correlations, PIP weights, use better normalization, remove separate treatment of FKP weights, etc.
Leaving this issue open til it's done, so that I remember to change this file in cosmopipe as well.

**Comment by pmcdonal** [comment]
Of course, if the randoms really exactly match the redshift evolution of the data as they're supposed to, you should get the same alpha with *any* z weighting (i.e., a *common* change in weighting shouldn't matter)... but might as well make sure to take out any error in the distribution as well as possible.
